### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Mengyao-Li** [Zhaoyi-Yan/Shift-Net_pytorch#77]
Hello,Regarding the dataset, do I put all the data directly in opt.dataroot, then the code will automatically divide the data set into a training set, a validation set, a test set? Or do I only put the training set into opt.dataroot?

**Comment by Zhaoyi-Yan** [comment]
Hi, you only need to set `opt.dataroot` to the folder of `training set`(when training) or `testing set`(when testing).

**Comment by Mengyao-Li** [comment]
Thanks for your reply.I have another problem,I interrupted the training. If I want to continue training, should I run python train.py --continue_train  or others?
I have run python train.py --batchsize=1 --use_spectral_norm_D=1 --which_model_netG='unet_shift_triple'  --model='shiftnet'  --shift_sz=1  --mask_thred=1 --mask_type='center'  --continue_train,but it start training again not continue training.

**Comment by Zhaoyi-Yan** [comment]
In fact, it actually continues training. It loads `latest` model then continues training with `epoch 1`. If you change `which_epoch=2`, then it continues training with `epoch 3`.
